tests/Mailjet/Message/EventNotificationMessageTest.php

```diff
@@ -5,12 +5,11 @@
 use AppBundle\Entity\Adherent;
 use AppBundle\Mailjet\Message\EventNotificationMessage;
 use AppBundle\Mailjet\Message\MailjetMessageRecipient;
-use Tests\AppBundle\Config;
 
 class EventNotificationMessageTest extends AbstractEventMessageTest
 {
-    const SHOW_EVENT_URL = 'https://'.Config::APP_HOST.'/comites/59b1314d-dcfb-4a4c-83e1-212841d0bd0f/evenements/2017-01-31-en-marche-lyon';
-    const ATTEND_EVENT_URL = 'https://'.Config::APP_HOST.'/comites/59b1314d-dcfb-4a4c-83e1-212841d0bd0f/evenements/2017-01-31-en-marche-lyon/inscription';
+    const SHOW_EVENT_URL = 'https://enmarche.dev/comites/59b1314d-dcfb-4a4c-83e1-212841d0bd0f/evenements/2017-01-31-en-marche-lyon';
```

Comment:
can you use the parameter here please.

Reply:
its a unit test. not a functional test, so no container here